exercise_perseus_render/assets/src/vue/perseus.vue

```diff
@@ -165,13 +191,47 @@
       checkAnswer() {
         if (this.itemRenderer) {
           const check = this.itemRenderer.scoreInput();
-          this.correct = check.correct;
           this.empty = check.empty;
+          if (!check.empty) {
+            this.complete = check.correct;
+            this.correct = this.hinted || !this.firstAttempt ? false : check.correct;
+            this.$parent.$emit('checkanswer', this.correct, this.complete, this.firstAttempt, this.hinted);
```

Comment:
Could you add a comment explaining what this line does?

Calling `this.$parent` is very unusual, let alone `this.$parent.$emit`.